health/notifications/alarm-notify.sh.in

```diff
@@ -2016,6 +2027,50 @@ EOF
   fi
 }
 
+
+# -----------------------------------------------------------------------------
+# Stackpulse sender
+
+send_stackpulse() {
+  local payload httpcode oldv currv
+  [ "${SEND_STACKPULSE}" != "YES" ] && return 1
+
+  # We are sending null when values are nan to avoid errors while JSON message is parsed
+  [ "${old_value}" != "nan" ] && oldv="${old_value}" ||  oldv="null"
+  [ "${value}" != "nan" ] && currv="${value}" || currv="null"
+
+  payload=$(cat <<EOF
+  {
+    "Node" : "${host}",
+    "Chart" : "${chart}",
+    "OldValue" : ${oldv},
+    "Value" : ${currv},
+    "Units" : "${units}",
+    "OldStatus" : "${old_status}",
+    "Status" : "${status}",
+    "Alarm" : "${name}",
+    "Date": ${when},
+    "Duration": ${duration},
+    "NonClearDuration": ${non_clear_duration},
+    "Description" : "${status_message}, ${info}",
+    "CalcExpression" : "${calc_expression}",
+    "CalcParamValues" : "${calc_param_values}",
+    "TotalWarnings" : "${total_warnings}",
+    "TotalCritical" : "${total_critical}",
+    "ID" : ${alarm_id}
+  }
+EOF
+)
+
+  httpcode=$(docurl -X POST -H "Content-Type: application/json" -d "${payload}" ${STACKPULSE_WEBHOOK})
+  if [ "${httpcode}" = "200" ]; then
+    info "sent stackpulse notification for: ${host} ${chart}.${name} is ${status}"
+  else
+    error "failed to send stackpulse notification for: ${host} ${chart}.${name} is ${status}, with HTTP response status code ${httpcode}."
```

Comment:
returni think `else` branch should `return 1`

Reply:
Yes, it should return 1, otherwise we'll always think we sent notifications successfully.

Reply:
Thank you guys! I inserted the return.